dolphinscheduler-task-plugin/dolphinscheduler-task-sqoop/pom.xml

```diff
@@ -48,5 +48,10 @@
             <artifactId>dolphinscheduler-datasource-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-service</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Please remove this module in task-plugin.

Reply:
> Please remove this module in task-plugin.

Hi @caishunfeng , thanks for your comment. 

This dependent is added in `dolphinscheudler-task-sqoop/pom.xml` since `sqoop` task needs to add its mask pattern through `SensitiveDataConverter.addMaskPattern()`, which is in `sdolphinscheduler-service` module.

<img width="378" alt="截屏2022-11-22 15 32 05" src="https://user-images.githubusercontent.com/38122586/203252444-09914e7e-ffe1-4249-a3ed-4f55ab55547b.png">

Is your suggestion that we should move `SensitiveDataConverter` to another module? (maybe `dolphinscheduler-task-api`?)

Reply:
> > Please remove this module in task-plugin.
> 
> Hi @caishunfeng , thanks for your comment.
> 
> This dependent is added in `dolphinscheudler-task-sqoop/pom.xml` since `sqoop` task needs to add its mask pattern through `SensitiveDataConverter.addMaskPattern()`, which is in `dolphinscheduler-service` module.
> 
> <img alt="截屏2022-11-22 15 32 05" width="378" src="https://user-images.githubusercontent.com/38122586/203252444-09914e7e-ffe1-4249-a3ed-4f55ab55547b.png">
> 
> Is your suggestion that we should move `SensitiveDataConverter` to another module? (maybe `dolphinscheduler-task-api`?)

Reply:
>Is your suggestion that we should move SensitiveDataConverter to another module? (maybe dolphinscheduler-task-api?)

Yes, is it better to move it inito common module? WDYT?

Reply:
> > Is your suggestion that we should move SensitiveDataConverter to another module? (maybe dolphinscheduler-task-api?)
> 
> Yes, is it better to move it inito common module? WDYT?

Sure, thanks for your suggestion, I'll look into it.